### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by Ibom99** [jupyterhub/outreachy#59]
### What page is this for?

https://ipython.org/usersurvey2013.html

### WAVE accessibility report

![Screenshot (236)](https://user-images.githubusercontent.com/104465078/195927416-05b03bf2-5eb5-4ee0-b8dc-b9b885a1b1ed.png)
![Screenshot (237)](https://user-images.githubusercontent.com/104465078/195927427-2eb5149a-15a0-4a1c-9473-7319253e3d8a.png)
WAVE Report of https://ipython.org/usersurvey2013.html including the evaluation and my suggestions.

### **Evaluation:**

**1. **(1)Error:(Language missing or invalid):**** This means the language of the document isn't identified or the `lang` attribute value is invalid. This affects how screen readers read the content of the page in the appropriate language. It also affects automatic translation of the content on the page.
 **Suggestion:** I would add the `lang="en"` attribute and value to the`<html></html>` tag, being that the page is in English. `"en"` is the valid value for a page with content in English.

**2. (35) Contrast Errors: (Very Low Contrast):** There is low contrast between the text color and it's background colors. High color contrast is very necessary for readability, especially for low vision users.
 **Suggestion:** For the first 17 contrast errors I will make the text colors a deeper red. The likes of maroon `#800000` or burgundy `#800020`. I would also increase the contrast by increasing the `font-weight` of the text. For the next contrast errors that are permalinks embedded in icons I would increase the `opacity` property of the icons to a value closer to 1 or their color to black `#000000`. For the copyright text in the footer section of the page I would change the text color to a darker gray `#696969`.

**3. ALERTS: Justified Text(42):** Fully justified text present. This affects readability of text due to the large spacing of words, as a result of `align:justify` styling on the text. 
**Suggestion:** I would remove the styling property from whichever tag has that property.

**4. ALERTS: Suspicious alternative text(9):** Alternative text is insufficient or contains vague text that isn't descriptive enough. Alternative text of an image is meant to be descriptive enough to aid screen readers have an idea of what the image is, or else that information wouldn't be available to them. It also assists in times when the images isn't available for whatever reasons.
**Suggestion:** 1st alert- for the logo of the IPython: Interactive Computing page `alt="Logo"` I would change the alt text to `alt="IPython Logo"`.
2nd alert- for the pie chart of result countries for spread data `alt="_images/countries1.png"` I would change the alt text to `alt="Pie Chart of the result countries from the spreadsheet cleaned data "`.
For the other `alt ` texts of the graphs, I would make them descriptive enough to describe the data they hold.

**5. ALERTS: Image with title(2):** The image has a title attribute and no alternative text. Alternative text provides a better accessibility and should be used. Whereas the title attribute will generate a mouse hover tooltip which more or may not be desired - this tooltip will not be presented to touch screen or keyboard users.
**Suggestion:** I would add an Alternative text to the images and describe them accordingly.

### The HTML element to be changed

```html
A.) <html xmlns="http://www.w3.org/1999/xhtml" style="margin-left: 380px;">...</html>
B.) <a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank">
Install
</a>
C.) <a class="reference external" href="https://docs.google.com/spreadsheet/pub?key=0AqIElKUDQl8tdHF2WmlKdTZTRlZVRGFGTDgtUXFBVUE&output=html">
a Google Spreadsheet
</a>
D.) <div class="footer" role="contentinfo">
© Copyright the IPython development team. Created using
Very low contrast
<a href="http://sphinx-doc.org/" style="color: rgb(136, 138, 133); background-color: rgb(255, 255, 255);">
Sphinx
</a>
E.) <img class="logo" src="_static/IPy_header.png" alt="Logo">
F.) <img alt="_images/countries1.png" src="_images/countries1.png" style="width: 697.5px; height: 459.0px;">
```

### Your proposed HTML (or CSS), after change

```html
A.) <html lang="en" xmlns="http://www.w3.org/1999/xhtml" style="margin-left: 380px;">...</html>
B.) CSS: a{
color: #800000; 
}
C.) CSS: a{
opacity: 0.8;
color: #000000;
}
D.) <div class="footer" role="contentinfo">
© Copyright the IPython development team. Created using
Very low contrast
<a href="http://sphinx-doc.org/" style="color:#696969 ; background-color: rgb(255, 255, 255);">
Sphinx
</a>
E.) <img class="logo" src="_static/IPy_header.png" alt="IPython Logo">
F.) <img alt="Pie Chart of the result countries from the spreadsheet cleaned data" src="_images/countries1.png" style="width: 697.5px; height: 459.0px;">
```

### More info

I would like to change the `<div>` for the footer section to a `<footer>` tag for a more semantic HTML5 codebase.

**Comment by minrk** [comment]
Thank you, those are good suggestions!

**Comment by Ibom99** [comment]
> Thank you, those are good suggestions!

Thank you for the review! @minrk

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
